docs/binder/governance.rst

```diff
@@ -124,6 +114,24 @@ Team roles
 There are no other specific roles, but we may revisit this in the
 future.
 
+A note on team membership and growth over time
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Being a team member is not a life sentence. We
+expect team members to teach and grow other members so that they can
+pass their organizational responsibilities on to them. This encourages
+us to increase diversity, reduce burn-out and allow team members to
+shift their focus in reaction to changing life circumstances.
+
+We ask that team members commit themselves in ~1 year cycles for new
```

Comment:
This came a bit out of the blue/like a footnote. What do you think about adding something like this around L59 where we talk about people spending a reasonable amount of time to help the project if they are team members. I think it would be good to mention there something like "team membership is for you if you think that over the next 12months you will be able to do this. Training and mentoring new team members is an investment by the existing team which means team membership requires commitment from both sides."

Though maybe we change this to 6months then? 12 sounds like a long time or like it could scare people because 12months is a long time. Though maybe in reality 12months will fly by?

My main point is that I think mentioning this "two way commitment" as well as the fact that this isn't for life here would be good.

Reply:
Yeah that's a fair point - lemme work it in earlier somehow